docs/3.0.0-beta.x/advanced/customize-admin.md

```diff
@@ -33,9 +33,101 @@ By default, the administration panel is exposed via [http://localhost:1337/admin
 
 The panel will be available through [http://localhost:1337/dashboard](http://localhost:1337/dashboard) with the configurations above.
 
+---
+
 ### Development mode
 
-**_Currently not available_**
+To enable the front-end development mode you need to start your application using the `--watch-admin` flag.
+
+```bash
+cd my-app
+strapi develop --watch-admin
+```
+
+With this option you can do the following:
+
+#### Customize the `strapi-admin` package
+
+All files added in `my-app/admin/src/` will either be replaced or added
+
+**Example: Changing the available locales of your application**
+
+```bash
+# Create both the admin and admin/src folders
+cd my-app && mkdir -p admin/src
+# Change the available locales of the administration panel
+touch admin/src/i18n.js
+```
+
+**Path: `my-app/admin/src/i18n.js**
+
+```js
+import { addLocaleData } from 'react-intl';
+import { reduce } from 'lodash';
+import en from 'react-intl/locale-data/en';
+import fr from 'react-intl/locale-data/fr';
+
```

Comment:
If I **copy/paste** this snippet code. I have an issue on `trads` variable.
I checked the origin the `import trads from './translations';`is missing.

Reply:
I have updated the example.